.storybook/themes/commerce.scss

```diff
@@ -884,6 +924,36 @@
 					max-width: var(--content-scale-width),
 					width: 100%,
 				),
+				right-rail: (
+					display: grid,
```

Comment:
In desktop we only need to define items that change from the mobile - `display: grid` and others below can be removed

Reply:
Of course, declaration has been removed from desktop settings